webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -709,7 +710,11 @@ public String generateLocalized(
 
         driver.addStep(new RawDocumentToFilterEventsStep());
         driver.addStep(new CheckForDoNotTranslateStep());
-        driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
+        if (bcp47Tag.equals("en-x-psaccent")) {
+            driver.addStep(new PseudoLocalizeStep());
+        } else {
+            driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
```

Comment:
@aurambaj I think the idea was to have this code and then create a new "pull command." We would then create a function in AssetClient (getPseudoLocalizedAssetForContent) to call this endpoint, /api/assets{assetId}/pseudo.